webapp/channels/src/components/advanced_create_post/advanced_create_post.tsx

```diff
@@ -1135,6 +1140,16 @@ class AdvancedCreatePost extends React.PureComponent<Props, State> {
         const messageIsEmpty = this.state.message.length === 0;
         const draftMessageIsEmpty = this.props.draft.message.length === 0;
 
+        // fix for FF not capturing the paste without formatting event when using ctrl|cmd + shift + v
+        if (e.key === KeyCodes.V[0] && e.metaKey) {
+            if (e.shiftKey) {
+                this.isNonFormattedPaste = true;
+                this.timeoutId = window.setTimeout(() => {
```

Comment:
Nothing to change here, but I wanted to mention that it's odd that this seems to be our best bet for fixing this. I searched around briefly, and I came across quite a few people running into a similar issue with Firefox in Google Sheets but no more information than that. I guess Firefox just behaves differently here

Reply:
yes, it does. I couldn't find something better or "less intrusive". This approach just  fix the particular issue for firefox and prevent as much as possible the risk of a weird regression.